src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -1383,16 +1384,16 @@
 				}
 			}
 
-			prototype._buildArcListviewSelection = function (page) {
-				// find or add selection for current list element
-				var arcListviewSelection = page.querySelector(selectors.SELECTION);
+			prototype._buildArcListviewElement = function (page, cclass) {
+				// find or add element for current list element
+				var arcListviewElement = page.querySelector("." + WIDGET_CLASS + cclass);
```

Comment:
In my opinion you don't need WIDGET_CLASS in this place.

Reply:
Right, removed.